apps/resources/metals-emacs.json

```diff
@@ -0,0 +1,12 @@
+{
+  "repositories": [
+    "central"
+  ],
+  "properties": {
+    "metals.client": "emacs"
```

Comment:
Does this translate to ` --java-opt -Dmetals.client=emacs`? I couldn't find a reference to `metals.client` in the generated binary when I tried installing it locally and I couldn't find docs on `properties` in the reference guide

https://get-coursier.io/docs/cli-install.html#application-descriptor-reference

Reply:
It's roughly equivalent, yes. When using `properties`, these are kept in `coursier/bootstrap/launcher/bootstrap.properties` in the bootstrap JAR, rather than passed to `java` in the shell preamble. The former is less likely to suffer from escaping issues. It also benefits from property substitution, like done [here](https://github.com/coursier/apps/blob/fb44ff33efe3f69183cfa6c5d547fc247dbbb744/apps/resources/mill.json#L11-L12), which is handled when the bootstrap starts.

Reply:
Makes sense. That’s very clever!